web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.MobileHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import { useLingui } from '@lingui/react';
+
+import HamburgerIcon from 'web-components/src/components/icons/HamburgerIcon';
+import { Subtitle } from 'web-components/src/components/typography/Subtitle';
+import UserAvatar from 'web-components/src/components/user/UserAvatar';
+import { cn } from 'web-components/src/utils/styles/cn';
+
+import { getDefaultAvatar } from 'pages/Dashboard/Dashboard.utils';
+
+import { UNNAMED } from './DashboardNavigation.constants';
+import { DashboardNavHeaderData } from './DashboardNavigation.types';
+
+type Props = {
+  activeAccount: DashboardNavHeaderData['activeAccount'];
+  onMenuClick: () => void;
+  mobileMenuOpen?: boolean;
+};
+
+export const DashboardNavigationMobileHeader = ({
+  activeAccount,
+  onMenuClick,
+  mobileMenuOpen = false,
+}: Props) => {
+  const { _ } = useLingui();
+  const { name, image } = activeAccount;
+
+  const avatarSrc =
+    image ||
+    getDefaultAvatar({
+      ...activeAccount,
+      type:
+        activeAccount.type === 'user'
+          ? (window as any).AccountType?.User || 'USER'
+          : (window as any).AccountType?.Org || 'ORG',
+    });
+
+  return (
+    <div
+      className={cn(
+        'md:hidden fixed top-0 w-[100%] z-40 flex gap-10 bg-bc-neutral-0 px-5 py-10 transition-all duration-300',
+        mobileMenuOpen && 'blur-sm',
+      )}
+      style={{ borderBottom: '1px solid #e5e7eb' }}
```

Comment:
let's use tailwind classes instead and avoir using raw color hex as this won't change with the theme

Reply:
you need to specific `border border-solid border-{color}`